frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -101,27 +101,30 @@ export class PieceEditor implements OnInit {
             (a, b) => a.order < b.order ? -1 : 1);
         let children: WorkingPiece[] = [];
         for (const position of this.displayedPieces[i].positions) {
-          const uuid = PieceApi.uuidFromRaw(position.siblingId);
+          const uuid = String(position.siblingId);
           const child = this.workingPieces.find(wp => wp.uuid === uuid);
           children.push(child);
         }
         this.displayedPieces.splice(i + 1, 0, ...children);
       }
     }
     this.renderingItems = this.displayedPieces.map(
-        (displayPiece, displayieceIndex) => {
-      const hasNoChidren = displayPiece.type === 'SEGMENT'
-          ? false : displayPiece.positions.length === 0;
-      const name = hasNoChidren ? '*' + displayPiece.name : displayPiece.name;
-      return {
-        name,
-        pieceIndex: displayieceIndex,
-        siblingId: displayPiece.siblingId,
-        type: displayPiece.type,
-        isOpen: displayPiece.isOpen,
-        uuid: displayPiece.uuid,
-      };
-    });
+        (displayPiece, displayieceIndex) =>
```

Comment:
Spelling: looks like 4 usages of `displayieceIndex`  should be `displayPieceIndex`